src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -659,8 +657,10 @@ function getServerPath(options: Options, platformInfo: PlatformInformation) {
     return serverPath;
 }
 
-function getServerFileName(platformInfo: PlatformInformation) {
-    const serverFileName = 'Microsoft.CodeAnalysis.LanguageServer';
+function getInstalledServerPath(platformInfo: PlatformInformation) {
```

Comment:
Are we moving to always stating return types for functions or not?

Reply:
we should, but I haven't enabled the rule, hence me making bad changes :) ill fix